.github/workflows/test_installer.yml

```diff
@@ -0,0 +1,63 @@
+name: Install Local Version of Opik
+
+on:
+    workflow_dispatch:
+
+jobs:
+    test_installation:
+        runs-on: ubuntu-20.04
+
+        steps:
+            - name: Checkout repo
+              uses: actions/checkout@v3
+              with:
+                ref: ${{ github.ref }}
+            
+            - name: Setup Python
+              uses: actions/setup-python@v5
+              with:
+                python-version: 3.12
+
+            - name: Install Opik
+              run: pip install sdks/python
+
+            - name: Install Test Dependencies
+              run: |
+                pip install -r tests_end_to_end/test_requirements.txt
+                playwright install
+
+            - name: Install Opik
+              run: |
+                cd deployment/docker-compose
+                docker compose up --detach
+
+            - name: Check Docker pods are up
```

Comment:
@AndreiCautisanu even if containers are up, it doesn't mean the application is ready..
`opik-backend`, for instance, is running db migrations, so this check is not enough.
In other cases, I would check http://localhost:5173 availability, but it will have the same result.
So here, you can just add a `sleep 30` before running your test

Reply:
@liyaka sleep isnt a good option. lets do a pod check. tests can do a busy wait. we need to avoid sleep

Reply:
It is definitely not ideal, but at the moment it's hard to check that backend is running.
Another option would be to run your test in the loop with timeout instead

Reply:
@liyaka @Nimrod007 ok if we add a sleep 30 before any tests after checking pods and change this approach as soon as we have a reliable way of checking for backend availability?